go/libkb/login_state.go

```diff
@@ -616,13 +623,16 @@ func (s *LoginState) passphraseLogin(lctx LoginContext, username, passphrase str
 		lks := NewLKSec(pps, res.uid, s.G())
 		secret, err := lks.GetSecret(lctx)
 		if err != nil {
-			return err
+			// Ignore any errors storing the secret.
+			s.G().Log.Debug("(ignoring) error getting lksec secret for SecretStore: %s", err)
```

Comment:
Just future-proofing.  If there's an error here in this convenience function, I don't want provisioning to stop.